server/src/main/scala/ch.epfl.scala.index.server/ApiImplementation.scala

```diff
@@ -74,7 +74,8 @@ class ApiImplementation(github: Github, userState: Option[UserState])(implicit v
           )
         )
       ).limit(1)
-    }.map(r => r.as[Project].headOption.map(hideId))
+//    }.map(r => r.as[Project].headOption.map(hideId))
+    }.map ( r => r.as[Project].headOption)
```

Comment:
@MasseGuillaume Maybe this will lead to some problems. I guess indexing all data